src/styles/_owncloud.scss

```diff
@@ -51,3 +51,4 @@
 @import 'theme/oc-modal';
 @import 'theme/oc-sidebar';
 @import 'theme/oc-radio';
+@import 'theme/oc-accordion';
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6407398)